python/lsst/ts/envsensors/device/vcp_ftdi.py

```diff
@@ -21,12 +21,14 @@
 
 __all__ = ["VcpFtdi"]
 
+import asyncio
 import logging
-from typing import Callable
+from typing import Callable, Tuple
 
-from pylibftdi import Device  # type: ignore
+from pylibftdi import Device, FtdiError  # type: ignore
```

Comment:
Why type: ignore? Is this the typing system whining about imports? If so, you might want to disable that warning in the package config, instead.

Reply:
Done, thanks for catching that.